Library/Output Templates/df_monster_template.html

```diff
@@ -0,0 +1,90 @@
+﻿<!DOCTYPE html>
+
+<!-- 
+
+  Dungeon Fantasy monster template by Steve Jackson Games.
+  Implemented by Slacker for use with Richard A. Wilkes' GCS.
+
+  Instructions: 
+	  - Melee weapons print their name, damage, and reach. Include all damage options in the damage field of the weapon. A thrusting broadsword's swung damage field would say "sw+1 cut or thr+2 imp", etc. Notes are not printed.
+	  - Ranged weapons print their name, damage, accuracy, range, rate of fire, shots, and bulk. See melee weapons for more details.
+	  - Special afflictions and such will generally need to be manually added in. You can usually use a melee weapon as a base with all relevant text in the damage field, then delete the rest for the final copy.
+	  - You must manually edit the .html file if you want to add a description to the monster, add block, add notes to DR, change style of parry, etc.
+		
+  Tips: 
+	  - DR is derived from the torso hit location. Add in locational DR to the notes.
+	  - If you do not want to include damage resistance in the traits list, give another advantage a DR bonus. Know that this will not properly reflect the point cost of the monster if you do.
+	  - Notes save their formatting. If you wish to remove that, use the key @ENCODING_OFF at the start of this template, before <!DOCTYPE html>.
+	  - You can delete the @TOTAL_POINTS line if you do not want it. I included it because I thought it would be useful for things like Ally. Use the default text template to output monster templates.
+
+-->
+
+<html>
+  <head>
+    <title>@NAME</title>
+		<style>
+	  	  html, body, #wrapper {
+	  	    width: 100%;
+	  	    height: 100%; 
+	  	  }
+	  	  #container, p, table {
+	  	    width: 480px;
+	  	  }
+	  	  #name {
+	  	    font-size: 14pt;
+	  	  }
+		</style>
+  </head>
+	
+  <body>
+	<div id="wrapper">
+	<div id="container">
+	  <table id="name">
+		  <col width="300">
+		  <tr>
+			  <td><b>@NAME</b></td>
+			  <td></td>
+			  <td><b>[@TOTAL_POINTS]</b></td>
+		  </tr>
+		</table>
+		
+		<p>Enter notes about the creature here.</p>
+
+		<table>
+		  <col width="160">
+			<tr>
+				<td><b>ST:</b> @ST</td>
+				<td><b>HP:</b> @BASIC_HP</td>
+				<td><b>Speed:</b> @BASIC_SPEED</td>
+			</tr>
+			<tr>
+				<td><b>DX:</b> @DX</td>
+				<td><b>Will:</b> @WILL</td>
+				<td><b>Move:</b> @BASIC_MOVE</td>
+			</tr>
+			<tr>
+				<td><b>IQ:</b> @IQ</td>
+				<td><b>Per:</b> @PERCEPTION</td>
+			</tr>
+			<tr>
+				<td><b>HT:</b> @HT</td>
+				<td><b>FP:</b> @BASIC_FP</td>
+				<td><b>SM:</b> @SIZE</td>
+			</tr></br>

```

Comment:
This extraneous `</br>` tag and the one on line 78 are what is causing the spurious rendering of the paragraph at the top of the page. Remove them and it will stop doing that.